### ISSUE TITLE
dealing Log Message in eventMacro

### ISSUE THREAD

**Issue by hubertyang88** [OpenKore/openkore#3348]
## ------------------ Openkore Issues Template ------------------

* **Openkore version git**: 

* **Server**: 

* **Bug Report / Feature Request**:
 I am trying eventMacro for its advanced function 
I found all kinds Msg like LocalMsg  SystemMsg , PubMsg  , but can't find a trigger like ‘console’ in macro to deal prompt text .
So , is there not an event ‘console’ in eventMacro?
 I read some source code of macro plugin , there is a sub checkConsole to deal console Msg , but I don't really understand how plugin works .

After some failed tryings,it turns out this feature is important for me to transform my macro to eventmacro , could you please add some code to deal "Log Message" ?
* **Summary**:

**Comment by alisonrag** [comment]
Nope, there is not a event in eventmacro to handle console messages.
It was developed like this to be smooth and fast

**Comment by hubertyang88** [comment]
I see, THX

**Comment by ya4ept** [comment]
https://openkore.com/wiki/PubMsg

**Comment by hubertyang88** [comment]
PubMsg is for someone speak in public , it will not be triggered by console Msg like "you are now attacking xxx" , Anyway, eventmacro is much better than macro.

**Comment by alisonrag** [comment]
for sure has a hook to what you are looking for.
if not exists, you can open a feature request so we can add to source code

**Comment by alisonrag** [comment]
can you print or paste what kind of message do you want to handle?

**Comment by hubertyang88** [comment]
Some error messages ,  such like" I lost my master " or "fail to attack" , some log message , such like message showed by "exp monster", those messages are not triggered by packets .
I understand eventMacro is all about event . Can an event triggered by what shows in consloe? or is it against the concept of eventMacro,cause this event will constantly checking?

**Comment by ya4ept** [comment]
@Henrybk
